src/test/java/com/courtcanva/ccfranchise/utils/OrderTestHelper.java

```diff
@@ -123,28 +127,50 @@ public static List<Order> AcceptedOrderList() {
         return orders;
     }
 
-    public static Order createOrder(String customerId, String postcode, Long totalAmount, Franchisee franchisee) {
+    public static Order mockAcceptedOrder1() {
```

Comment:
seems like what you changed is adding a `mockAcceptedOrder1()`, but the whole file messed up.
It's better to leave other code alone, and write your code at the bottom of file.